report.html/index.test.js

```diff
@@ -0,0 +1,98 @@
+describe('gulp-spawn-mocha tests', function () {
```

Comment:
I would prefer just one additional test which uses sinon to check that a fake stream was written to.